src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -1465,6 +1508,24 @@ public boolean unsolvedAndCalledByASimpleClassName(MethodCallExpr method) {
     }
   }
 
+  /**
+   * Check whether a field, invoked by a simple class name, is unsolved
+   *
+   * @param field the field to be checked
+   * @return true if the field is unsolved and invoked by a simple class name
+   */
+  public boolean unsolvedFieldCalledByASimpleClassName(FieldAccessExpr field) {
+    try {
+      field.resolve();
+      return false;
+    } catch (UnsolvedSymbolException e) {
+      // this check is not very comprehensive, since a class can be in lowercase, and a method or
+      // field can be in uppercase. But since this is without the jar paths, this is the best we can
+      // do.
+      return Character.isUpperCase(field.getScope().toString().charAt(0));
```

Comment:
I don't fully understand why being uppercase tells us whether the field is unsolved. Is there some other invariant here that I'm missing, such as that unsolved fields always are static and need to be accessed via a class name?

Reply:
Professor, I will rename it. We know the field is unsolved because in the visit method for `FieldAccessExpr`, we check to see if the field is solved first before coming to this check.

Reply:
I'm still not fully happy with this, but I'll let it go for now as tech debt given the need to move quickly